### PR TITLE
Reaper Scans new site

### LINKED ISSUE #13639: Reaper Scans manhwas not updating
### Source information and language

Reaperscans 1.3.31

### Steps to reproduce

both global update and individual manhwa updates don't work. Comics stuck on latest entries.

Example: on site "The great mage returns after 4000 years" goes by chapter 144 and in app still gets chapter 143.

### Expected behavior

Update chapter list

### Actual behavior

chapter list not updated

### Tachiyomi version

Stable 0.13.6

### Android version

Android 9

### Other details

Reaper scans site has weird cache issues

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by AriaMoradi**
Closes #13639
Closes #13731

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by henrik9999** [comment]
this completly removes the other reaperscans langauges is that intended?

**Comment by AriaMoradi** [comment]
> this completely removes the other reaperscans languages is that intended?

Other languages don't exist right now...

**Comment by henrik9999** [comment]
> > this completely removes the other reaperscans languages is that intended?
> 
> Other languages don't exist right now...

what is that supposed to mean

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 1]
```diff
@@ -17,7 +17,6 @@ class MadaraGenerator : ThemeSourceGenerator {
         MultiLang("MangaForFree.net", "https://mangaforfree.net", listOf("en", "ko", "all"), isNsfw = true, className = "MangaForFreeFactory", pkgName = "mangaforfree", overrideVersionCode = 1),
         MultiLang("Manhwa18.cc", "https://manhwa18.cc", listOf("en", "ko", "all"), isNsfw = true, className = "Manhwa18CcFactory", pkgName = "manhwa18cc", overrideVersionCode = 2),
         MultiLang("Olympus Scanlation", "https://olympusscanlation.com", listOf("es", "pt-BR")),
-        MultiLang("Reaper Scans", "https://reaperscans.com", listOf("en", "fr", "id", "tr"), className = "ReaperScansFactory", pkgName = "reaperscans", overrideVersionCode = 7),
```
Comment (Riztard) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:20]: the other lang is still working btw
also why remove without replace

**Comment by henrik9999** [comment]
@AriaMoradi i have created a pr to your repo which adds search

**Comment by alessandrojean** [comment]
Just a side note for future similar cases like this one: as the English website isn't using the Madara code at all, it should have been moved out to an individual extension in `/src/en/reaperscans/`, exactly like it was before with the other languages Reaper have that used other themes.

**Pull request merged**